qiita_db/portal.py

```diff
@@ -5,13 +5,66 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
+import warnings
 
 from .sql_connection import SQLConnectionHandler
 from .util import convert_to_id
+from .exceptions import QiitaDBError, QiitaDBDuplicateError
 from qiita_core.qiita_settings import qiita_config
+from qiita_core.exceptions import IncompetentQiitaDeveloperError
 
 # Only make functions available for main portal
 if qiita_config.portal == "QIITA":
+    def create_portal(portal, desc):
```

Comment:
However, I think this is another discussion and we should move it outside of this PR if we want to continue talking about this.